### PR TITLE
Adding new databases support

### PR DESCRIPTION
**Pull request by tjlee**
Added:
- sample db dumps for supported dbs
- docker-compose entry for derby
- docker-compose entry for h2 1.3
- docker-compose entry for sybase 15.7

### CODE REVIEW

**Comment by tjlee** [comment]
In progress...

File: .env [hunk 1]
```diff
@@ -14,3 +14,7 @@ ORACLE_PASSWORD=guest
 # for db2
 DB2INST1_PASSWORD=root_pass
 LICENSE=accept
+
+# sybase user/pwd
+SYBASE_USER=sa
```
Comment (ignatov) [inline .env:19]: is it possible to use guest/guest credentials?

Comment (tjlee) [reply .env:19]: I'm not sure for sybase, need to investigate more how to rewrite default credentials

File: docker-compose.yml [hunk 2]
```diff
@@ -114,3 +114,13 @@ h2:
     - "9092:9092"
   env_file: .env
 
+
+hsql23:
+  image: chernov/hsqldb
+  ports: 
+    - "9001:9001"
+  env_file: .env
+
```
Comment (ignatov) [inline docker-compose.yml:123]: empty lines

File: docker-compose.yml [hunk 3]
```diff
@@ -92,3 +92,20 @@ db2:
...
   env_file: .env
+
+sybase:
+  image: chernov/sybase_15_7:latest
+  ports:
+    - "5000:5000"
+  env_file: .env
+  hostname: dksybase
+
+#derby:
```
Comment (ignatov) [inline docker-compose.yml]: ??

**Pull request closed without merging**